packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2101,7 +2101,7 @@ export const audiusBackend = ({
         }
         return {
           type: NotificationType.Milestone,
-          entityType: Entity.Playlist,
+          entityType: data.is_album ? Entity.Album : Entity.Playlist,
```

Comment:
yea it's kind of a confusing state because not all notif types have is_album but i'll update for it to be optional. we actually getting the album entity type downstream here: https://github.com/AudiusProject/audius-protocol/blob/357b2eb56aadecba31f4b21cc64ff0ce2d8cefa0/packages/mobile/src/screens/notifications-screen/Notifications/RepostNotification.tsx#L49-L54